server/routes/api/v1/admin/surveys.js

```diff
@@ -187,60 +187,41 @@ router.get('/csv', auth.ensureLoggedIn, auth.ensureAdmin, function(req, res) {
       });
     });
   } else if (include === 'last' || include === 'first') {
+      var order = include
+    var surveyCountParam = ids.length - 1;
+
+    var q = knex.select(knex.raw('subquery.*, a.value, q.text'))
+        .from(function(){
+            this.select(knex.raw('s.name, c.user_id, c.id as completion_id, ( select count(c2.id) from completions c2 where c2.user_id = c.user_id) as survey_count'))
+                .from(knex.raw('completions c'))
+                .orderBy('c.created_at', order)
+                .joinRaw('left outer join surveys s on c.survey_id = s.id')
+                .whereIn(knex.raw('s.id'), ids)
+                .as('subquery')
+        })
+        .joinRaw('left outer join answers a on subquery.completion_id = a.completion_id')
+        .joinRaw('left outer join questions q on q.id = a .question_id')
+        .whereRaw('survey_count > ?', surveyCountParam )
+        .whereIn(knex.raw('a.question_id'), qids)
+
+
+    q.then(function(resp) {
```

Comment:
This is a bit of a regression because we lose the 2nd row of headers that have the question id.